src/zmq/zmqnotificationinterface.cpp

```diff
@@ -72,7 +72,7 @@ CZMQNotificationInterface* CZMQNotificationInterface::CreateWithArguments(const
 // Called at startup to conditionally set up ZMQ socket(s)
 bool CZMQNotificationInterface::Initialize()
 {
-    LogPrint("zmq", "Initialize notification interface\n");
+    LogPrint("zmq", "zmq: Initialize notification interface\n");
```

Comment:
Right, this could be done by `LogPrint` (autoprint the category). But until then, it is probably best if we keep it consistent with other `LogPrint`'s.